api/handlers/container/start.go

```diff
@@ -17,7 +19,26 @@ import (
 func (h *handler) start(w http.ResponseWriter, r *http.Request) {
 	cid := mux.Vars(r)["id"]
 	ctx := namespaces.WithNamespace(r.Context(), h.Config.Namespace)
-	err := h.service.Start(ctx, cid)
+
+	detachKeys := r.URL.Query().Get("detachKeys")
```

Comment:
Can we add some validation for possible detachkeys values?

Reply:
According to this documentation user can provide an arbitrary sequence of their choice to detached from the container - https://docs.docker.com/reference/cli/docker/container/attach/#detach-keys

I can add some sort of validation to check if the sequence they've provided is in the appropriate format

Reply:
Added validation